Source/WebKit/NetworkProcess/cache/NetworkCacheDataCocoa.mm

```diff
@@ -90,7 +90,8 @@
 
 Data Data::subrange(size_t offset, size_t size) const
 {
-    return { adoptOSObject(dispatch_data_create_subrange(dispatchData(), offset, size)) };
+    RetainPtr data = dispatchData();
```

Comment:
We should introduce a `protectedDispatchData()` if we don't already have one.

Reply:
Did not think of that, makes sense.